aclk/README.md

```diff
@@ -1,17 +1,15 @@
 <!--
----
 title: "Agent-Cloud link (ACLK)"
 description: "The Agent-Cloud link (ACLK) is the mechanism responsible for connecting a Netdata agent to Netdata Cloud."
-date: 2020-04-30
+date: 2020-05-11
 custom_edit_url: https://github.com/netdata/netdata/edit/master/aclk/README.md
----
 -->
 
 # Agent-cloud link (ACLK)
 
 The Agent-Cloud link (ACLK) is the mechanism responsible for securely connecting a Netdata Agent to your web browser
-through Netdata Cloud. The ACLK is encrypted, safe, and _does not exchange data with Netdata Cloud until you claim a
-node_.
+through Netdata Cloud. The ACLK establishes an outgoing WebSocket connection to Netdata Cloud on port `443`. The ACLK is
+encrypted, safe, and _does not exchange data with Netdata Cloud until you claim a node_.
```

Comment:
does not exchange data with Netdata Cloud until you claim a node
->
is only established if you claim the node.

This way, it sounds like it's established and not sending data, which isn't accurate